lib/App/DuckPAN/Cmd/Server.pm

```diff
@@ -190,11 +190,13 @@ sub change_js {
 }
 
 # Rewrite all relative asset links in CSS
-# E.g url("/assets/background.png") => url("http://duckduckgo.com/assets")
+# Capture leading quote, insert $hostname, append filename
+# E.g url("/assets/background.png") => url("http://duckduckgo.com/assets/background.png")
 sub change_css {
 	my ( $self, $css ) = @_;
 	my $hostname = $self->hostname;
-	$css =~ s!:\s*url\((["'])?/!:url\($1http://$hostname/!g;
+
+	$css =~ s|:\s*url\(["']([^'"]+)["']\)|:url\("http://$hostname/$1")|gi;
```

Comment:
The problem was related to our parsing and re-writing of relative urls in the CSS files we pull down from duckduckgo.com.

After looking into this more, I've realized that re-writing the CSS isn't even necessary because Web.pm proxies relative urls to the appropriate upstream host.

Removing the body of this function has no effect on the SERP AFAICT.

I will remove these changes